src/main/resources/interface-specification.yml

```
@@ -6230,6 +6230,104 @@ paths:
               schema:
                 type: integer
               description: Time interval in milliseconds. Allowed requests will be constantly replenished during the interval. At the end of the interval the max allowed requests will be available
+  /clients/{clientId}/purposes/{purposeId}:
+    parameters:
+      - $ref: '#/components/parameters/CorrelationIdHeader'
+      - $ref: '#/components/parameters/IpAddress'
+    delete:
+      tags:
+        - clients
+      summary: Removes a purpose from a client
+      description: Removes a purpose from a client
+      operationId: deleteClientPurpose
```

may we stick with the current approach and use `remove` instead of `delete`?

```suggestion
      operationId: removeClientPurpose
```